scripts/zones/East_Ronfaure/npcs/Geomantic_Reservoir.lua

```diff
@@ -0,0 +1,31 @@
+-----------------------------------
+-- Area: East Ronfaure
+-- NPC: Geomantic Reservoir
+-- Unlocks Geo-Poison
+-- !pos 378.875 -39.304 58.313
+-----------------------------------
+local ID = require("scripts/zones/East_Ronfaure/IDs")
+local entity = {}
+
+entity.onTrade = function(player, npc, trade)
+end
+
+entity.onTrigger = function(player, npc)
+    if player:hasSpell(769) then -- Player has Indi-Poison
```

Comment:
Please use our enums instead of naked magic numbers:
https://github.com/LandSandBoat/server/blob/base/scripts/globals/spell_data.lua#L1054

Example here:
https://github.com/LandSandBoat/server/blob/base/scripts/quests/hiddenQuests/Trust_Ingrid.lua#L18